protos/openoffload.proto

```diff
@@ -98,7 +98,7 @@ rpc getClosedSessions(statisticsRequestArgs) returns (stream sessionResponse) {}
 // or have the applications have a mechanism to avoid duplicate sessionIDs across 
 // applications since there will be many applications instances to 1 switch
 message sessionId{
-  int64 sessionId = 1;
+  uint64 sessionId = 1;
```

Comment:
Why did u change this to uint64 ? I thought we were reserving -1 etc for indicating error conditions and such

Reply:
nvm - this was simply making them consistent across the two message.